gnovm/tests/files/assign33.gno

```diff
@@ -0,0 +1,13 @@
+package main
+
+func foo() (int, bool) {
+	return 1, true
+}
+
+func main() {
+	var a, b int = foo()
+	println(a, b)
+}
+
+// Error:
+// main/files/assign33.gno:8:6: cannot use bool as int
```

Comment:
the err message can be improved some what, like in go: "cannot use foo() (value of type bool) as int value in assignment"

Reply:
Fixed